apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -145,6 +141,45 @@ export class JestPlugin implements IHeftPlugin {
     clean.properties.pathsToDelete.add(cacheFolder);
   }
 
+  private _getJestReporters(
+    heftSession: HeftSession,
+    heftConfiguration: HeftConfiguration
+  ): (string | Config.ReporterConfig)[] {
+    const config: Config.GlobalConfig = JsonFile.load(this._getJestConfigFolder(heftConfiguration));
+
+    if (Array.isArray(config.reporters)) {
+      // Replace the 'default' reporter with the heft reporter
+      const defaultIndex: number = config.reporters.indexOf('default');
+      if (defaultIndex >= 0) {
+        config.reporters[defaultIndex] = this._getHeftJestReporterConfig(heftSession, heftConfiguration);
+      }
+
+      return config.reporters;
+    } else {
+      // Otherwise if no reporters are specified install only the heft reporter
+      return [this._getHeftJestReporterConfig(heftSession, heftConfiguration)];
+    }
+  }
+
+  private _getHeftJestReporterConfig(
+    heftSession: HeftSession,
+    heftConfiguration: HeftConfiguration
+  ): Config.ReporterConfig {
+    const reporterOptions: IHeftJestReporterOptions = {
+      heftConfiguration,
+      debugMode: heftSession.debugMode
+    };
+
+    return [
+      path.resolve(__dirname, 'HeftJestReporter.js'),
+      (reporterOptions as unknown) as Record<string, unknown>
+    ];
+  }
+
+  private _getJestConfigFolder(heftConfiguration: HeftConfiguration): string {
```

Comment:
`JEST_CONFIGURATION_LOCATION` points to the Jest config file, not the config folder, so this name should be updated.

Reply:
Can you also update [line 44](https://github.com/microsoft/rushstack/pull/2290/files#diff-68f293fb9db78cdc60e2df2de0f6eeb54dc1e1e34b47dbfa5ce784e58aa5fd5bR44) to use this function?